priv/repo/migrations/20231114150716_add_products_table.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Phil.Repo.Migrations.AddProductsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:products) do
+      add :name, :string, null: false
+      add :ticket_type_id, :integer, null: false
```

Comment:
I never remember should it be `:bigint` or does ecto do that for us. Also, I'd probably make it `:text` unless we need to order by that column?